modules/030-cloud-provider-azure/docs/CONFIGURATION.md

```diff
@@ -22,6 +22,8 @@ StorageClass Configuration Parameters:
 
 * `provision` — sets additional StorageClasses for [Azure ultra disks](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#ultra-disk):
   * `name` — the name of the class to create;
+  * `type` — SKU type;
+  * `cachingMode` — string value that corresponds to destired caching mode;
```

Comment:
Provide information about available values. Add link to the docs https://docs.microsoft.com/en-us/azure/virtual-machines/premium-storage-performance#disk-caching.

Reply:
Done.